step-core/src/main/java/step/core/artefacts/reports/ReportNodeStatus.java

```diff
@@ -18,8 +18,14 @@
  ******************************************************************************/
 package step.core.artefacts.reports;
 
+// ATTENTION: The order is important here, lower ordinals have higher "importance".
+// Keep RUNNING as the last ine in the definitions.
 public enum ReportNodeStatus {
-	
+
+	VETOED,
+
+	IMPORT_ERROR,

```

Comment:
These 2 new statuses are actually not related to report nodes and thus shouldn't be declared here. The issue is that we're using the root report node status as result of the execution. This is actually wrong and should be changed in the future. As this might be a larger refactoring I'm approving this PR for now and we should refactor this later